scripts/gateway-systemd-install.sh

```diff
@@ -27,14 +29,15 @@ Documentation=https://www.firezone.dev/kb
 
 [Service]
 Type=simple
+Environment="FIREZONE_VERSION=$FIREZONE_VERSION"
```

Comment:
Is this line needed? I don't think the service needs this env var?

Reply:
@jamilbk this is so that a specific version can be installed, I also decided to keep it here so that we can log it if needed